test/common/models/Mailbox.js

```diff
@@ -144,8 +150,12 @@ describe('The Mailbox class', function() {
       var mailbox = jmap.Mailbox.fromJSONObject({}, {
         id: 'id',
         name: 'name',
+        namespace: {
+          type: 'personal'
+        },
         parentId: 'parentId',
         role: 'inbox',
+        sharedWith: {},
```

Comment:
No test or business logic for `shareWith` property yet ? Indeed, it involves a new data structure and some upcoming new models.

Reply:
Exactly, we will see that after.